### DISCUSSION TITLE
Problems running model "ggml-vicuna-13b-1.1-q4_0" getting error.

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#1590] [Q&A]
I am trying to run a model in oobabooga named "ggml-vicuna-13b-1.1-q4_0" 

I am getting the error 

Traceback (most recent call last):
  File "F:\AI2\oobabooga-windowsBest\text-generation-webui\server.py", line 914, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "F:\AI2\oobabooga-windowsBest\text-generation-webui\modules\models.py", line 132, in load_model
    from modules.llamacpp_model_alternative import LlamaCppModel
  File "F:\AI2\oobabooga-windowsBest\text-generation-webui\modules\llamacpp_model_alternative.py", line 9, in <module>
    from llama_cpp import Llama, LlamaCache
ImportError: cannot import name 'LlamaCache' from 'llama_cpp' (F:\AI2\oobabooga-windowsBest\installer_files\env\lib\site-packages\llama_cpp\__init__.py)

I might need to install some package etc. I installed oobabooga and updated it via pip. But I have not really done anything else. Most models work for me. But I have yet to get a ggml working. Thank you.

**Comment by 1980Dragon** [comment]
Same error here. 
In my case, it was a version issue with llama-cpp-python.

Run cmd_windows.bat in oobabooga_windows folder
pip uninstall llama-cpp-python
pip install llama-cpp-python 

I succeeded in running ggml models using this method.

**Reply by viperwasp** [reply]
Thanks but I think that is still a no go. I did this in both the main oobabooga folder and inside text-generation-webui. 
And my command line is "python server.py --auto-devices --chat --model ggml-vicuna-13b-1.1-q4_0 --wbits 4 --groupsize 128 --model_type llama"

Starting the web UI...
Gradio HTTP request redirected to localhost :)
Loading ggml-vicuna-13b-1.1-q4_0...
Traceback (most recent call last):
  File "F:\AI2\oobabooga-windowsBest\text-generation-webui\server.py", line 914, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "F:\AI2\oobabooga-windowsBest\text-generation-webui\modules\models.py", line 132, in load_model
    from modules.llamacpp_model_alternative import LlamaCppModel
  File "F:\AI2\oobabooga-windowsBest\text-generation-webui\modules\llamacpp_model_alternative.py", line 9, in <module>
    from llama_cpp import Llama, LlamaCache
ImportError: cannot import name 'LlamaCache' from 'llama_cpp' (F:\AI2\oobabooga-windowsBest\installer_files\env\lib\site-packages\llama_cpp\__init__.py)

**Reply by 1980Dragon** [reply]
oobabooga-windows\installer_files\env\bin\llama.dll
copy this file to 
oobabooga-windows\installer_files\env\lib\site-packages\llama_cpp <== this folder
if its not there already. 

If this method doesn't work, unfortunately, I have no further advice to offer. Good luck.

**Reply by viperwasp** [reply]
Thanks I did not have llama.dll located in my oobabooga-windows\installer_files\env\bin\llama.dll but it was already located at oobabooga-windows\installer_files\env\lib\site-packages\llama_cpp. I even tried putting it in both locations. Still not working. Thanks for trying to help. If I ever get it resolved I will post here.

**Reply by Javier-Machin** [reply]
Any luck with this @viperwasp ?

**Reply by viperwasp** [reply]
No I don't think I made progress.
